docs/cpu_templates/cpu-templates.md

```diff
@@ -6,6 +6,8 @@ to the guest software by changing the following configuration:
 - CPUID (x86_64 only)
 - MSRs (Model Specific Registers, x86_64 only)
 - ARM registers (aarch64 only)
+- Vcpu feaures (aarch64 only)
```

Comment:
We need to update the changelog to mention the new feature.
nit: s/Vcpu/vCPU/

Reply:
Added note to the CHANGELOG